### ISSUE TITLE
Problems about the CelebA dataset

### ISSUE THREAD

**Issue by LGYoung** [hubert0527/COCO-GAN#16]
Hi,
I also try to test your model on my costume dataset.
One question about your testing procedure:
Do you split the CelebA( or CelebA-64*64) dataset for training and testing, and precalculate the fid-stat on the testing part?
Or do you pre-calculate the fid-stat on the whole dataset, train the model using the whole dataset, and compare the statistics of fake samples with the pre-calculated fid-stat?

**Comment by hubert0527** [comment]
At the time of research, I do split train-test sets, train on training set then test on the test set. But recent research in GANs does not really need that, since the train/test distribution difference is really small, and the FID does not have significant differences.
